monitoring/uss_qualifier/scenarios/astm/netrid/display_data_evaluator.py

```diff
@@ -306,11 +346,20 @@ def _evaluate_normal_observation(
                 pass  # TODO: Check position, altitude, flight details, etc
 
     def _evaluate_area_to_large_observation(
```

Comment:
nit: Unrelated to your changes but it seems to be a typo in the function name (`too` instead of `to` large?)

Reply:
Good catch -- fixed